substrate/frame/multisig/src/lib.rs

```diff
@@ -186,6 +186,18 @@ pub mod pallet {
 		Multisig<BlockNumberFor<T>, BalanceOf<T>, T::AccountId, T::MaxSignatories>,
 	>;
 
+	/// A mapping containing information on whether a multisig has an expiry
+	/// time or not.
+	#[pallet::storage]
+	pub type MultisigExpiries<T: Config> = StorageDoubleMap<
+		_,
+		Twox64Concat,
+		T::AccountId,
+		Blake2_128Concat,
+		[u8; 32],
```

Comment:
I think it is better to use H256 for hash type

Reply:
Maybe, but we already use `[u8; 32]` in the `Multisigs` storage map.

Reply:
Is there any reason why we would want to have the same here as in the `Multisigs` storage map?

Reply:
The only reason I have for this is to keep things consistent in the pallet. But I can make it a `H256` if you think so.

Reply:
If it's better, then sure